pkg/controllers/vdb/createdb_reconciler.go

```diff
@@ -129,9 +129,20 @@ func (c *CreateDBReconciler) execCmd(ctx context.Context, initiatorPod types.Nam
 	if res, err := c.Dispatcher.CreateDB(ctx, opts...); verrors.IsReconcileAborted(res, err) {
 		return res, err
 	}
+	passwd := ""
+	clusterPodRunner, ok := c.PRunner.(*cmds.ClusterPodRunner)
+	if ok {
+		passwd = clusterPodRunner.VerticaSUPassword
+	} else {
+		fakePodRunner, ok2 := c.PRunner.(*cmds.FakePodRunner)
+		if ok2 {
+			passwd = fakePodRunner.VerticaSUPassword
+		}
+	}
+
 	if c.Vdb.IsCertRotationEnabled() {
 		cmd := []string{
-			"-f", PostDBCreateSQLFileVclusterOps,
+			"--password", clusterPodRunner.VerticaSUPassword, "-f", PostDBCreateSQLFileVclusterOps,
```

Comment:
Should here use passwd`?
```suggestion
			"--password", passwd "-f", PostDBCreateSQLFileVclusterOps,
```

Reply:
@LiboYu2 -- if you don't plan to take the suggestion from a comment, can you reply saying why you don't take it instead of simply ignore the comment?

Or have you already addressed this?

Reply:
> @LiboYu2 -- if you don't plan to take the suggestion from a comment, can you reply saying why you don't take it instead of simply ignore the comment?
> 
> Or have you already addressed this?

Reverted the original change. There is a change in main branch to use PRunner.ExecVSQL to execute the SQL script. That function already takes password into consideration.